acceptance/bundle/deploy/volume/recreate/databricks.yml.tmpl

```diff
@@ -0,0 +1,31 @@
+bundle:
```

Comment:
Is this a new file or a copy from integration tests? Are there more tests that depend on it in intergation tests?

Ideally it would be a move like in other PRs, easier to review.

Reply:
good catch! removed the original template in the follow-up pr